bin/crew

```diff
@@ -757,22 +763,17 @@ def prepare_package (destdir)
 
     # check for conflicts with other installed files
     conflicts = []
-    Dir[CREW_META_PATH + '*.filelist'].each do |packageList|
-      packageName = File.basename packageList, '.filelist'
-      File.readlines(packageList).each do |line|
-        File.readlines('filelist').each do |item|
-          if line == item && packageName != @pkg.name
-            conflicts << [packageName, item.chomp]
-          end
-        end
-      end
-    end
+    @conflictscmd = "grep --exclude #{CREW_META_PATH}#{@pkg.name}.filelist \
+      -Fxf filelist #{CREW_META_PATH}*.filelist | tr ':' ' ' | \
+      sed 's,.filelist,,g' | sed 's,#{CREW_META_PATH},,g'"
+    conflicts << %x[#{@conflictscmd}].chomp.split(" ")
+    conflicts.reject!(&:empty?)
     unless conflicts.empty?
       puts "Unable to complete this build since there is a conflict with the same file in another package.".lightred
       pp = ''
       conflicts.each do |p, f|
         puts "\n#{p} package conflicts below:".lightred if p != pp
-        puts f.lightred
+        puts "#{f}".lightred
```

Comment:
This line doesn't need to be changed.

Reply:
I know, but it was throwing errors unless I did that. :/

Reply:
If it actually works for you feel free to change that back...

Reply:
works now